x-pack/test_serverless/api_integration/test_suites/common/reporting/generate_csv_discover.ts

```diff
@@ -478,6 +478,8 @@ export default ({ getService }: FtrProviderContext) => {
     });
 
     describe('nanosecond formatting', () => {
+      // failsOnMKI, see https://github.com/elastic/kibana/issues/179456
+      this.tags(['failsOnMKI']);
```

Comment:
I think this should go to the top level suite in this file as we also saw other steps being flake.